[DBM] PR feedback on #15900 (#16016)
DataDog/documentation: layouts/shortcodes/dbm-sqlserver-agent-config-examples.md

    custom_queries:
    - query: SELECT age, salary, hours_worked, name FROM hr.employees;
      columns:
        - name: sqlserver.custom.employee_age
          type: gauge
        - name: sqlserver.custom.employee_salary
           type: gauge
        - name: sqlserver.custom.employee_hours
           type: count
        - name: name
           type: tag
      tags:
        - 'table:employees'
```
### Working with hosts through a remote proxy

> If the Agent must connect to a database host through a remote proxy, all telemetry will be tagged with the hostname of the proxy rather than the database instance. Utilize the `reported_hostname` option to set a custom override of the hostname detected by the agent.

Reviewer: ```suggestion
If the Agent must connect to a database host through a remote proxy, all telemetry is tagged with the hostname of the proxy rather than the database instance. Utilize the `reported_hostname` option to set a custom override of the hostname detected by the Agent.
```
